ui.js

```diff
@@ -150,6 +150,9 @@ function handleDemoDropdown() {
       case "Ocean":
         monacoEditor.setValue(oceanDemoCode);
         break;
+      case "Image":
```

Comment:
Maybe call this demo "Texture from URL".

Reply:
Changed